app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Help and advice
+navigation: 50
+image: "media/images/content/hero-images/0025.jpg"
+subtitle: We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
```

Comment:
@gazcarr Not a biggie for the prototype but just to flag that the subtitle font size is much bigger on this Help and advice page compared with the Train to be a teacher page, plus the images are different heights